rally-domain/src/main/groovy/yakworks/rally/mail/EmailUtils.groovy

```diff
@@ -24,21 +24,27 @@ class EmailUtils {
      * Examples: <br>
      * "joe@email.com" - simple single email <br>
      * "Account Services <tesla@greenbill.io>, "Galt, John" <galt@yak.works>, joe@email.com" - 3 valid emails
-     * @return true is valid
-     * @throw ThrowableProblem with code:validation.problem
+     *
+     * @throws ThrowableProblem if validation fails
      */
-    static boolean validateEmail(String emails) {
-        boolean isValid = false
+    static void validateEmail(String emails) {
+        if(!emails?.trim()) {
+            throw Problem.of("validation.problem").detail("Empty email").toException()
+        }
         try {
             InternetAddress[] addys = InternetAddress.parse(emails)
-            addys.each{
-                it.validate()
+            for (InternetAddress addr : addys) {
+                try {
+                    addr.validate()
+                } catch (AddressException e) {
+                    throw Problem.of("validation.problem")
```

Comment:
<img width="364" alt="image" src="https://github.com/user-attachments/assets/7c2e8b35-c98b-4436-9e16-22647ef9034b">

has a specific meaning. expects name to be passed. or we can consider using DataProblem.
Also below, comment on why your adding payload reference to the InternetAddress.

Reply:
Fixed payload, changed to use DataProblem